pkgs/top-level/all-packages.nix

```diff
@@ -20522,10 +20522,14 @@ with pkgs;
   };
 
   glfw = glfw3;
-  glfw-wayland-minecraft = callPackage ../development/libraries/glfw/3.x-wayland-minecraft.nix {};
   glfw2 = callPackage ../development/libraries/glfw/2.x.nix { };
-  glfw3 = callPackage ../development/libraries/glfw/3.x.nix {
+
+  glfw3 = callPackage ../by-name/gl/glfw3/package.nix {
+    inherit (darwin.apple_sdk.frameworks) Carbon Cocoa Kernel;
+  };
```

Comment:
```suggestion
```
We could probably use the darwin scope directly in the expression, as most packages in pkgs/by-name don't handle `darwin.apple_sdk` in this way anymore